quarterback-packages/track-changes-plugin/src/utils/track-utils.ts

```diff
@@ -39,6 +39,6 @@ export function createNewUpdateAttrs(
   return {
     ...attrs,
     operation: CHANGE_OPERATION.set_node_attributes,
-    oldAttrs: restAttrs,
+    oldAttrs: JSON.parse(JSON.stringify(restAttrs)),
```

Comment:
why do we stringify and then parse object?

Reply:
to deepclone it, alternatively can add a lodash dependency to use deepClone from there, but it's not that big of an object to justify it, i think. What do you think?

Reply:
that is cool. on need to add lodash